examples/plot_simulate_mpi_backend.py

```diff
@@ -63,7 +63,8 @@
 # ``openmpi``, which must be installed on the system
 from hnn_core import MPIBackend
 
-with MPIBackend(n_procs=2, mpi_cmd='mpiexec'):
+n_procs = 1
```

Comment:
think we should revert this to `n_procs=2` as it was before? not sure if that was causing the CircleCI issue

Reply:
Will do. In fact, #272 passed just fine with python=3.6, so I'll revert it here too. I'm hoping it's just been a fluke. I really don't see how this PR could have broken MPI...